build/queryOverview.js

```diff
@@ -203,7 +203,7 @@ var queryOverview = (function (webcharts) {
     status_order: ["Open", "Answered", "Closed", "Cancelled"],
     site_col: "Site Name",
     groups: null, // array of objects with value_col/label properties
-    filters: [{ value_col: "Query Open By: Marking Group", label: "Marking Group" }],
+    filters: [],
```

Comment:
Going to go ahead and dismiss the review and merge since @samussiah is on vacay 🏄🏾